DMTN-107.tex

```diff
@@ -67,14 +68,18 @@ \subsection{Alert Production}\label{ssec:req_ap}
 
 \subsection{Alert Packet Contents}\label{ssec:req_apc}
 
-The Data Products Definition Document \citeds{LSE-163} specifies that the alert packets include {\em at least} a 12 month history, and cutouts of the template and difference images (Section 3.5.1).
-The 12 month history is not a true requirement, and should be interpreted as applying only after 12 months of LSST operations.
+\citeds{LSE-61} contains a formal requirement that the alert packets include {\em at least} a 12 month history (if available), as well as cutouts of the template and difference images\dmreq{0274}.
+The 12 month history would not be available in LOY1, and the requirement adequately accounts for this, so it would not be violated in LOY1.
 Alerts issued without image cutouts would not violate any requirements, but would be a non-standard packet. 
 
-\citeds{LSE-61} contains a formal requirement that the variability characterization parameters in an alert packet include data collected during {\em at least} the past year\reqparam{diaCharacterizationCutoff}\dmreq{0319}.
+\citeds{LSE-61} contains a formal requirement that the variability characterization parameters in an alert packet be based on data collected during {\em at least} the past year\reqparam{diaCharacterizationCutoff}\dmreq{0319}.
 In LOY1, releasing variability characterization parameters in the alerts that have been calculated with $<1$ year of data would, technically, be in violation of this requirement.
 However, that is unavoidable no matter which option for alert production in LOY1 is adopted.
 
+\citeds{LSE-61} also contains a formal requirement that all DIA sources in the Prompt Products database have a cross-match positional association with the Data Release objects catalog\dmreq{0324}.
+Since this association is part of the DIA source record, it would also be included in the alert packet \citeds{LSE-163}.
```

Comment:
That's not correct; it's anticipated to be a separate table/view.  I'm sorry if I produced this misunderstanding.  I.e. the requirement applies to "Alert Production" (the process) but not "Alert" content.

Reply:
Ah, whoops. OK. Replacing the second item (line 80) with these:
\citeds{LSE-163} furthermore states that the alert packet contents will contain the identification of the matched DR Object (Section 3.5.1; but this is not a formal requirement).
Prior to the generation of DR1 in LOY1, this requirement will not be met, and DR Object catalog identifications will not be included in alerts.

Reply:
@gpdf discussed this in person.  If the point of this sentence is to emphasize that the linkage to the DR Object is included in the alert packet, it is sufficient to replace DIASource record with DIAObject recrod, as the linkage is required there (DMS-REQ-0271)

Reply:
I'm sorry, this is still a tangle.

It *is* a requirement (per DMS-REQ-0271) for the *DIAObject* record (which by DMS-REQ-0274 is part of the Alert Content) to contain links to 3+3 nearby *Objects*, as well as the content (presumably including IDs) of 12 months of *DIASources*. 

But it does not contain a *full* positional *DIASource* to *Object* mapping.

Reply:
Agreed, sufficient to use DMS-REQ-0271. New version will say: 
\citeds{LSE-61} contains a formal requirement that all DIA objects shall include identifiers of the nearest three stars and three galaxies from the Data Release Objects catalog\dmreq{0271}.
\citeds{LSE-163} furthermore states that the alert packet contents will contain these identifiers (Section 3.5.1; however, this is not a formal requirement).
Prior to the generation of DR1 in LOY1 this requirement cannot be met, and the alert packets will not contain associations to LSST DR Objects. 
As in the above paragraph, this is unavoidable no matter which option for alert production in LOY1 is adopted.

Reply:
Sorry, one more thing: I struck the DPDD statement about the DR Object IDs in DM-16059 entirely (since the object ids are included in the DIAObject record).  So I would stick to the LSE-61 requirement (DMS-REQ-0274) to include the complete DIAObject record, and avoid the DPDD mention entirely.

Reply:
Is the DM-16059 change still waiting to go into the released DPDD?  The ticket is marked "done" but there's no LCR link.

Reply:
Specifically I would phrase as 
\citeds{LSE-61} contains a formal requirement that all DIAObjects shall include identifiers of the nearest three stars and three galaxies from the Data Release Objects catalog\dmreq{0271}, and these DIAObjects will be included in the alert packet\dmreq{0274}.
Prior to the generation of DR1 in LOY1 this requirement cannot be met, and the alert packets will not contain associations to LSST DR Objects.
As in the above paragraph, this is unavoidable no matter which option for alert production in LOY1 is adopted.

Reply:
@gpdf It's pending on master for a future batch LCR.

Reply:
Will adopt @ebellm 's wording, thank you!